dask_cuda/local_cuda_cluster.py

```diff
@@ -163,7 +163,7 @@ def __init__(
         if n_workers is None:
             n_workers = len(CUDA_VISIBLE_DEVICES)
         self.host_memory_limit = parse_memory_limit(
-            memory_limit, threads_per_worker, n_workers
+            memory_limit, threads_per_worker, n_workers or 1
```

Comment:
This is just to avoid a divide by zero error when creating `LocalCUDACluster` with `n_workers=0`. Which is probably a rare situation.

Reply:
Makes sense to me, I still don't understand why would anyone startup a `LocalCUDACluster` with 0 workers though, is there more to it besides your test case? If you're only creating the cluster for the scheduler, wouldn't it make more sense to test creating a scheduler and then a worker instead of `LocalCUDACluster` with 0 workers?

Reply:
Yeah I don't see this happening except for my test case.

Reply:
Sorry @jacobtomlinson , probably my fault this has stalled, but after taking another look at it, maybe we should change to `n_workers=1` in the test, and then `await client.wait_for_workers(2)`, rather than leaving room for users referring to an unsupported behavior. As a bonus, we could check and error when `n_workers < 1`. What do you think?

Reply:
Thanks @pentschev that sounds reasonable.

Updated to reflect this.